man/docker-exec.1.md

```diff
@@ -28,6 +29,9 @@ container is unpaused, and then run
 **-d**, **--detach**=*true*|*false*
    Detached mode: run command in the background. The default is *false*.
 
+**--detach-keys**=[]
+   Define the key sequence to use to detach from the container.
```

Comment:
Using `from` seems awkward without saying what's being detached from what. In "Define the key sequence which detaches from the container", it sounds like the key sequence is being detached from the container.